packages/component-library/components/Spacing/padding.ts

```diff
@@ -0,0 +1,85 @@
+import { GridFractions, Spacing } from "./types"
+import { convertFractionToString } from "./util"
+
+const styles = require("./Padding.module.scss")
+
+export const pt = (unit: GridFractions): string[] => [
+  styles[`pt-${convertFractionToString(unit)}`],
+]
+
+export const pr = (unit: GridFractions): string[] => [
+  styles[`pr-${convertFractionToString(unit)}`],
+]
+
+export const pb = (unit: GridFractions): string[] => [
+  styles[`pb-${convertFractionToString(unit)}`],
+]
+
+export const pl = (unit: GridFractions): string[] => [
+  styles[`pl-${convertFractionToString(unit)}`],
+]
+
+export const px = (unit: GridFractions): string[] => [
+  styles[`pl-${convertFractionToString(unit)}`],
+  styles[`pr-${convertFractionToString(unit)}`],
+]
+
+export const py = (unit: GridFractions): string[] => [
+  styles[`pt-${convertFractionToString(unit)}`],
+  styles[`pb-${convertFractionToString(unit)}`],
+]
+
+export const p = (unit: GridFractions): string[] => {
+  const classes: string[] = []
+  if (typeof unit === "number") {
+    classes.push(styles[`p-${convertFractionToString(unit)}`])
+  } else {
+    Object.keys(unit).forEach(key => {
+      classes.push(styles[`p-${convertFractionToString(unit[key])}--${key}`])
+    })
+  }
+
+  return classes
+}
+
+export const responsivePaddingClasses = ({
```

Comment:
A bit of a nitpick, but perhaps this should be renamed to just `paddingClasses` until we tackle responsitivity again, to avoid confusion.